src/mam/mod_mam_pm.erl

```diff
@@ -673,8 +672,7 @@ hooks(HostType) ->
         {amp_determine_strategy, HostType, fun ?MODULE:determine_amp_strategy/3, #{}, 20},
         {sm_filter_offline_message, HostType, fun ?MODULE:sm_filter_offline_message/3, #{}, 50},
         {get_personal_data, HostType, fun ?MODULE:get_personal_data/3, #{}, 50}
-        | mongoose_metrics_mam_hooks:get_mam_hooks(HostType)
-    ].
```

Comment:
Why this change? The previous version would technically be just as correct and also generate less intermediate garbage.

Reply:
Under the hood, the generated opcode would be the same. Dunno why I changed it, gonna revert it back.